### PR TITLE
Footer design updates

### PR DESCRIPTION
**Pull request by davidrapson**
This PR updates the footer to match the latest designs. Dark style, matching theguardian.com

screen shot 2015-07-09 at 15 05 23

@tudorraul @afiore

### CODE REVIEW

**Comment by davidrapson** [comment]
@tudorraul This look OK to you?

File: assets/stylesheets/_vars.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
 $guss-colours: $pasteup-palette;
 
 $c-brand: guss-colour(guardian-brand);
-$c-brand-mid: #005083;
+$c-brand-tint: #025b8c;
```
Comment (tudorraul) [inline assets/stylesheets/_vars.scss]: is this not accessible via `guss-colour`?

**Comment by tudorraul** [comment]
:+1: bar the question

File: assets/stylesheets/_vars.scss (hunk 1 above)
Comment (davidrapson) [inline assets/stylesheets/_vars.scss]: Nope, frontend use a lightening function base on the brand colour. But I just hardcoded, might just try that.

**Pull request merged**